backend/engine/utils/plugin.py

```diff
@@ -8,12 +8,11 @@
 from urllib.parse import quote_plus
 
 import boto3
-from botocore.exceptions import ClientError
-from django.db.models import Q
-
 from artemisdb.artemisdb.models import PluginConfig
 from artemislib.logging import Logger
 from artemislib.util import dict_eq
+from botocore.exceptions import ClientError
+from django.db.models import Q
```

Comment:
These should be grouped as before, where the 3rd party imports are together.

Reply:
Hmm, `black` is doing that automatically. I may need to tweak how I have it configured with VSCode.

Regardless, should be fixed now.

Reply:
I have had trouble recently with `black` and `isort` import sorting in VSCode so I've turned autosorting off.